packages/theme/src/lib/light-theme.ts

```diff
@@ -816,6 +816,9 @@ export const lightTheme = {
         color: 'secondary',
       },
       styleOverrides: {
+        root: {
+          padding: '.75rem',
+        },
```

Comment:
would it make more sense to just get rid of the size prop if they want both sizes to be medium styling?